nutrition_labels/grant_tagger.py

```diff
@@ -230,7 +241,15 @@ def fit(self, X, y):
         self.model = model.fit(X, y)
 
     def predict(self, X):
-        return self.model.predict(X)
+        y_predict = self.model.predict(X).astype(int)
```

Comment:
this is not needed if the user has set the threshold so I would add it as an else statement after
```
if self.pred_prob_threshold:
    # If the prediction probability is over a threshold then allow a 1
    # prediction to stay as 1, otherwise switch to 0.
    # A prediction of 0 will stay at 0 regardless of probability.
    pred_probs = self.model.predict_proba(X)
    y_predict = y_predict*(np.max(pred_probs, axis=1) >= self.pred_prob_threshold
else:
    y_predict = self.model.predict(X).astype(int)
```